.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
it seems a bit unnecessary to run on all python versions x all host platforms, but it doesn't seem to punish us (yet) for CI time, and the test suites seem fast enough.